src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -119,7 +119,7 @@ public DynamoAnalyticsClient(HostAnalyticsInfo hostAnalyticsInfo)
             Session.Start();
 
             //Dynamo app version.
-            var appversion = DynamoModel.AppVersion;
+            var appversion = string.IsNullOrEmpty(hostAnalyticsInfo.AppVersion) ? DynamoModel.AppVersion : hostAnalyticsInfo.AppVersion;
```

Comment:
to me this seems to indicate we should just remove AppVersion and make all integrators use HostAnalyticsInfo? Unless we are being cautious here because of the timing?

Reply:
Actually yes I think we should remove it. ( we have too many versions in the model anyway ). But we still want that default value - I'll bring it to where is needed.